TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
This would add a new dependency to the `TYPO3.Neos.NodeTypes` Package, right? Not sure if we should add also a default for the default Page/Document or but this content into that prototype?